<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="tree_file" default="test_tree.xml"/>
    <node name="magnetometer_compass" pkg="compass" type="magnetometer_compass"
          respawn="true" respawn_delay="1.0" output="screen">
        <rosparam subst_value="true">
            publish_mag_azimuth_ned_red: true
        </rosparam>
    </node>
    <node name="test_tree" pkg="road_crossing" type="run_tree"
          respawn="true" respawn_delay="1.0" output="screen">
        <rosparam subst_value="true">
            tree_file: $(arg tree_file)
        </rosparam>
    </node>
</launch>